src/Features/Core/Portable/Debugging/AbstractBreakpointResolver.cs

```diff
@@ -227,8 +227,12 @@ private async Task<IEnumerable<INamedTypeSymbol>> GetAllTypesAsync(CancellationT
         return namespaces.GetAllTypes(cancellationToken);
     }
 
-    private static IMethodSymbol GetPartialImplementationPartOrNull(ISymbol symbol)
-        => (symbol.Kind == SymbolKind.Method) ? ((IMethodSymbol)symbol).PartialImplementationPart : null;
+    private static ISymbol GetPartialImplementationPartOrNull(ISymbol symbol) => symbol.Kind switch
```

Comment:
cc @tmat  to take a look. Not familiar with debugger, but I feel this change brings no harm.